src/testUtils/getEffects.js

```diff
@@ -0,0 +1,17 @@
+import { isEqual } from 'lodash';
+/**
+ * getEffect(prereqs, reactModule)
+ * Returns the callbacks passed to useEffect mock call with the given prerequisites key,
+ * or null if none match.
+ *
+ * @param {any[]} prereqs - unique prerequisite list identifying the target useEffect call
+ * @param {object} reactModule - react module to be mocked.
+ * @return {function} - callback method passed for the effect hook.
+ */
+const getEffects = (prereqs, reactModule) => {
```

Comment:
why passing react module as arguments? Can we just create a method that `setupTest.js` can invoke to mock React?

Reply:
Alas, this repo will not run setupTest against the parent, and attempting to mock from the local version of React did not override for the tests.

Reply:
but that requires it hit the whole repo at once